### ISSUE TITLE
Discovery Ticket: Review steps for 'Downtime Notifications' page (content edits for #73181)

### ISSUE THREAD

**Issue by johnny-jesensky-adhoc** [department-of-veterans-affairs/va.gov-team#73191]
### Context
Suggested content edits from [footer ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/73181) 

### Description

>"Please review the list order here: https://depo-platform-documentation.scrollhelp.site/developer-docs/downtime-notifications#Downtimenotifications-Formslibraryapplications
> 
> It appears that (3)(a)(c) should really be (3)(b), and that (3)(b) should be (3)(b)(a) and (3)(c) should be (3)(b)(b). That is, one of the optional parameters for downtime is requiredForPrefill, and that parameter can take arguments of true or false.
> 
> If this isn't the case, the list should be revised to make the usage of downtime more clear."

### Tasks
- [x] Review footer, Github ticket
- [x] Confirm suggested edits on steps from a Platform engineer
- [x] Document findings from this ticket in another ticket, which should be linked here.

### Acceptance Criteria
- [x] Page edits have been confirmed with a Platform engineer
- [x] Necessary edits should be logged, documented in a separate ticket.

**Comment by johnny-jesensky-adhoc** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Platform-Content-Team-62bde00be46b400017674aef/issues/department-of-veterans-affairs/va.gov-team/73191?planning-poker) with Zenhub @BSmolensky @laucon @taylojill

**Comment by johnny-jesensky-adhoc** [comment]
Need to make separate ticket to perform changes after discovery has been completed.

**Comment by jknipes** [comment]
@BSmolensky please review

**Comment by BSmolensky** [comment]
Contacted last editor of page (Theo Bentum) via Slack. Theo edited the pages all the way back in 2022. So, hoping Theo is still the SME for this ticket. Will update with more when available.

**Comment by BSmolensky** [comment]
Pinged Above contact this morning. Still waiting on response.

**Comment by BSmolensky** [comment]
Theo has sent me to Curt Bonade. I have reached out to him and will report back when I hear from him.

**Comment by BSmolensky** [comment]
Curt has sent me to Nick Sprinkle. Waiting to hear from Nick now.

**Comment by BSmolensky** [comment]
First contact with Nick was yesterday at 4pm. I pinged him again this morning at 11:15am. Still waiting for reply.

**Comment by BSmolensky** [comment]
Heard back from Nick. He said: "Our program does own the forms library, but I have to be honest that I've never used that downtime functionality before or even seen this code haha. I guess you could ask someone like Robin Garrison or maybe Micah Chiang as they've been working on production forms for a long time and may have used this before?" 

But he added: "Again, I've never used it, but when I read those steps nothing stands out to me as being out of order or out of place...just based on what I know about prefill"

I asked him to clarify what the last statement meant and he said: "what already exists on the page looks right."

This means there is a good chance that the ticket originator is incorrect and that what exists is already correct. 

I have reached out to both Robin and Micah. Hope to hear something before the end of the sprint. But it looks like we might be able to close this discovery ticket without creating an edit ticket. Because nothing needs to chance. More to come. 

NOTE: I will be on PTO Monday 4/8. But will be still have Tuesday and Wednesday to close this out. Hopefully.

**Comment by jknipes** [comment]
Discovery completed, creating separate edit ticket. Moving this ticket to _Done_

**Comment by BSmolensky** [comment]
Contacts for this ticket recommend contacting someone on the Dev Ops or Backend Team to act as the SME on the actual edit ticket.
